turf-swift.podspec

```diff
@@ -6,8 +6,8 @@ Pod::Spec.new do |s|
   s.version = "0.0.3"
```

Comment:
[The marketing version in Info.plist](https://github.com/mapbox/turf-swift/blob/master/Turf/Info.plist#L18) should match this version.